src/clock.c

```diff
@@ -116,5 +117,21 @@ rt_tick_t rt_tick_from_millisecond(rt_int32_t ms)
 }
 RTM_EXPORT(rt_tick_from_millisecond);
 
+/**
+ * This function provides a tick value ALWAYS in millisecond
+ *
+ * @return 1ms-based tick
+ */
+RT_WEAK rt_tick_t rt_hw_1ms_tick_get(void)
```

Comment:
=> `RT_WEAK int rt_tick_get_millisecond(void);` to get the passed millisecond from boot.

Reply:
ok